src/main/java/sirius/biz/jobs/batch/file/ArchiveExportJob.java

```diff
@@ -79,5 +83,28 @@ protected void collectParameters(Consumer<Parameter<?, ?>> parameterCollector) {
             super.collectParameters(parameterCollector);
         }
     }
+
+    /**
+     * Digests every entry of a fresh created export archive.
+     * <p>
+     * Override this method in order to perform validations on contents of the final archive.
+     *
+     * @param digester a consumer receiving the name and the {@link InputStream} of each entry
+     */
+    protected void digestExportedFile(BiConsumer<String, InputStream> digester) {
```

Comment:
why should this be overwritten?
when is this invoked where?

Reply:
As the method name says, this method provides access to the InputStream of the fresh exported file (does not matter which type: could be a CSV, or a ZIP containing several files). Since getting access to this stream is not always needed, I find it more elegant that the caller explicitly does it if needed. For this there are different methods depending on the parent class (FileExportJob or ArchiveExportJob). The wording validation could be somewhat misleading I agree... is just the typical use case why one needs this InputStream.

XMLExportJob is one place that makes usage of this digester, in order to run the XSD validation on the export file.